.travis.yml

```diff
@@ -13,5 +13,4 @@ install:
 script:
   - go get -t -v ./...
   - diff -u <(echo -n) <(gofmt -d -s .)
-  - go tool vet .
```

Comment:
`go test` runs a subset of of vet checks, so it’s still useful to run vet. It should be done this way now:

```
- go vet ./...
```